5주차 과제 고양이 장난감가게 유효성 추가하기, 회원 관리 만들기

jihwooon opened the pull request:
이번주 5주차 과제 올립니다.

한 주 동안 잘 부탁 드립니다~!!

지난 주는 많이 못 올렸지만 이번주는 힘내서 열심히 해보도록 하겠습니다!

jihwooon committed:
5주차 과제 고양이 장난감가게 유효성 추가하기, 회원 관리 만들기

이번주 5주차 과제 올립니다.

한 주 동안 잘 부탁 드립니다~!!

지난 주는 많이 못 올렸지만 이번주는 힘내서 열심히 해보도록 하겠습니다!

jihwooon committed:
Add : Dozer mapper 추가

Gradle 에 추가
implementation 'com.github.dozermapper:dozer-core:6.4.0'

hannut91 commented on app/src/main/java/com/codesoom/assignment/App.java:
```diff
@@ -1,15 +1,21 @@
+    @Bean
+    public Mapper dozerMapper() {
+        return DozerBeanMapperBuilder.buildDefault();
+
```
0여기 빈 줄이 들어갔네요

hannut91 commented on app/src/main/java/com/codesoom/assignment/application/ProductService.java:
```diff
@@ -12,9 +14,11 @@
     private final ProductRepository productRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    public ProductService(Mapper dozerMapper,ProductRepository productRepository) {
```
```suggestion
    public ProductService(Mapper dozerMapper, ProductRepository productRepository) {
```

여기 빈 칸이 들어가야 할 것 같아요. IDEA의 자동 정렬 사용을 습관하 하시면 좋을 것 같아요

jihwooon replied:
띄어쓰기, 빈칸 된 줄 알았는데 의외로 빈 곳이 있네요... 
IDEA 자동 정렬 단축키 : macOS: Cmd + Alt + L

hannut91 reviewed (commented):
이번 한 주도 잘 부탁드립니다 ㅎㅎ

jihwooon committed:
Add : UserController 추가

UserService 개발
UserControllerTest 개발

hannut91 commented:
굉장히 작은 단계부터 하나씩 기능을 만들고 계신 것 같아요. 잘하고 계신 것 같습니다 ㅎㅎ 과제를 완성해 보세요

jihwooon committed:
Fix : UserTest Builduer 변경

creationWithBuilder 변경
changeWithUser 변경

jihwooon commented:
> 굉장히 작은 단계부터 하나씩 기능을 만들고 계신 것 같아요. 잘하고 계신 것 같습니다 ㅎㅎ 과제를 완성해 보세요

작은 단위라도 테스트 진행하면서 계속해서 구현해 보겠습니다~!

hannut91 reviewed (commented):
과제풀이를 보고 과제를 완성해 보세요ㅎㅎ

jihwooon committed:
Add : User 회원 가입 구현하기

UserRepository, UserController, User Service 구현하기

UserService Test 완료
UserController Test 완료

hannut91 commented:
이번 한 주도 고생 많으셨습니다. 이미 지난 일은 뒤로 하고, 회고를 통해서 앞으로 어떻게 나아갈 지 고민해 보시면 좋을 것 같아요.

jihwooon committed:
Add : 과제 풀이 1 고양이 장남감가게 유효성 추가, 회원관리 만들기 과제풀이

완성.....
